Flutter/lib/JoinSettle.dart

```diff
@@ -1,12 +1,9 @@
-import 'package:flutter/material.dart';
-import 'package:Settle/LobbyScreen.dart';
-import 'Server.dart';
-import 'Settle.dart';
-import 'app_localizations.dart';
+import 'import_all.dart';
```

Comment:
Again, importing everything does not seem like a good idea

Reply:
Resolved in commit `app-style 89939db`